Automattic/woocommerce-payments
# Use CardNotice component for authorization notification in transaction details page

Fixes #7748 

#### Changes proposed in this Pull Request

* Use recently added CardNotice component for authorization notification in transaction details page.











**Before**

<img width="1191" alt="284339226-b7c44c40-222c-46c9-a58d-c17d7873cbba" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/bfc7833f-39d4-42b6-9d62-d7d952a21240">


**After**

<img width="1063" alt="Screenshot 2023-12-12 at 19 05 43" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/aa77fdb2-1353-4701-8c0d-7c759673f144">

#### Testing instructions





1. As a merchant, go to Payments > Settings and enable Issue an authorization on checkout, and capture later
2. As a shopper, purchase an order. Take note of the order to use it in the next step.
3. As a merchant, navigate to Payments > Transactions > Uncaptured tab
4. Find the Authorization that matches your order and enter its details

To test an authorization that is blocked by fraud rules you can force `isFraudOutcomeReview` flag. In `client/payment-details/summary/index.tsx`:

```javascript
const isFraudOutcomeReview = true;
``` 
5. Refresh the page.
6. The capture button should not be present and the UI should match the following screenshot:

_Authorization blocked by fraud rules_

<img width="1120" alt="Screenshot 2023-12-13 at 14 05 59" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/9a54f477-3837-429f-9270-6c6ed355973e">


-------------------


**Post merge**
